### ISSUE TITLE
Add: Library Management Categories

### ISSUE THREAD

**Issue by ckxu** [TDXDigital/TPS#65]
Government Category (CRTC category; ie: Cat21, Cat32, etc.)

Genre (Playlist category)
Categories:
Rock/Pop
Folk/Roots
Electronic
Experimental
Jazz/Classical
Hip-hop
Heavy/Punk/Metal
World

Blocking Category (Daytime1 [06:00-12:00], Daytime2 [12:00-18:00], Evening [18:00-00:00], Nighttime [00:00-06:00])

**Comment by FireByTrial** [comment]
do these these Key/Value pairs work:
```php
"govCats"=>array(
                    // CRTC Categories http://www.crtc.gc.ca/eng/archive/2010/2010-819.HTM
                    "11" => "News",
                    "12" => "Spoken word-other",
                    "21" => "Pop, rock and dance",
                    "22" => "Country and country-oriented",
                    "23" => "Acoustic",
                    "24" => "Easy listening",
                    "31" => "Concert",
                    "32" => "Folk and folk-oriented",
                    "33" => "World beat and international",
                    "34" => "Jazz and blues",
                    "35" => "Non-classic religious",
                    "36" => "Experimental Music",
                    "41" => "Musical themes, bridges and stingers",
                    "42" => "Technical tests",
                    "43" => "Musical station identification",
                    "44" => "Musical identification of announcers, programs",
                    "45" => "Musical promotion of announcers, programs",
                    "51" => "Commercial announcement",
                    "52" => "Sponsor Identification",
                    "53" => "Promotion with sponsor mention",
                ),
                "genres"=>array(
                    "RK-P" => "Rock/Pop",
                    "FO-R" => "Folk/Roots",
                    "EL"   => "Electronic",
                    "EX"   => "Experimental",
                    "JZ-C" => "Jazz/Classical",
                    "HH"   => "Hip-Hop",
                    "HM"   => "Heavy/Punk/Metal",
                    "WLD"  => "World",
                ),
                "Block"=>array(
                    "D1" => "Daytime1  [06:00-12:00]",
                    "D2" => "Daytime2  [12:00-18:00]",
                    "E0" => "Evening   [18:00-00:00]",
                    "LN" => "Nighttime [00:00-06:00]",
                ),
```

**Comment by ckxu** [comment]
I think it looks good. The genres might correspond to folders though, so
may be best to limit them to 2 chars each. Example:

"genres"=>array(
                    "RP" => "Rock/Pop",
                    "FR" => "Folk/Roots",
                    "EL" => "Electronic",
                    "EX" => "Experimental",
                    "JC" => "Jazz/Classical",
                    "HH" => "Hip-Hop",
                    "HM" => "Heavy/Punk/Metal",
                    "WD" => "World",

What do you think?


On Wed, Aug 5, 2015 at 1:59 AM, James <notifications@github.com> wrote:

> Closed #65 <https://github.com/TDXDigital/TPS/issues/65> via e94fc99
> <https://github.com/TDXDigital/TPS/commit/e94fc99c9008bc13ac37fcb0eb7ae8b6763a57f8>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TDXDigital/TPS/issues/65#event-373911707>.
>



-- 
*Sean Warkentine,*

*Music Director at CKXU 88.3 FM*

*twitter.com/CKXUmusic <http://twitter.com/CKXUmusic>*
*facebook.com/CKXUmusic <http://facebook.com/CKXUmusic>*
*instagram.com/CKXUmusic <http://instagram.com/CKXUmusic>*
Listen at 88.3 FM or online at *www.ckxu.com <http://www.ckxu.com>*

-- 

*CKXU Radio Society*
SU162, 4401 University Drive West, Lethbridge, Alberta, T1K3M4
www.ckxu.com
